### DISCUSSION TITLE
Unpickling error??? i think??

### DISCUSSION THREAD

**Discussion by mimikwoomy** [voicepaw/so-vits-svc-fork#411] [Q&A]
hhelpp
![Screenshot (683)](https://user-images.githubusercontent.com/50461448/233198291-505f2c14-333c-40d2-aad9-96aa6f7730f0.png)

**Comment by Qualzz** [comment]
same. Did you use svc pre-hubert -fm crepe ?

**Reply by mimikwoomy** [reply]
No, I used my own svc and dio iirc
